### PR TITLE
Aligns language server with protocol proposition

### PR DESCRIPTION
**Pull request by giltho**
Aligns the server with microsoft/language-server-protocol#953

### CODE REVIEW

File: jsonrpc/src/main.ts [hunk 1]
```diff
@@ -269,7 +269,7 @@ export interface LogTraceParams {
 }
 
 export namespace LogTraceNotification {
-	export const type = new NotificationType<LogTraceParams, void>('$/logTraceNotification');
+	export const type = new NotificationType<LogTraceParams, void>('logTrace');
```
Comment (dbaeumer) [inline jsonrpc/src/main.ts:272]: Actually we should name this `$/logTrace` since clients and servers don't need to implement this.

**Comment by dbaeumer** [comment]
@giltho should we for consistency change this one as well?

https://github.com/microsoft/vscode-languageserver-node/blob/master/jsonrpc/src/main.ts#L262

**Comment by giltho** [comment]
Yes I think it would be better indeed. Also, I could probably add it to the protocol proposition.

**Comment by dbaeumer** [comment]
That would be great!

**Comment by giltho** [comment]
I updated the draft here microsoft/language-server-protocol#953 :)

**Comment by dbaeumer** [comment]
@giltho could you resolve the conflict please.

**Comment by giltho** [comment]
@dbaeumer done :)

**Pull request merged**